book/visual-effects.md

```diff
@@ -1596,9 +1597,7 @@ After drawing all of the element contents with `cmds` (and applying
 opacity), this code draws a rounded rectangle on another layer to
 serve as the mask, and uses destination-in blending to clip the
 element contents. Here I chose to draw the rounded rectangle in white,
-but the color doesn't matter as long as it's opaque. On the other
-hand, if there's no clipping, I don't round the corners of the mask,
-which means nothing is clipped out.
+but the color doesn't matter as long as it's opaque.
```

Comment:
The deleted sentence literally isn't true any more, if there's no clipping we don't mask anything at all (instead of just not rounding the corners, which was an earlier draft.)